### ISSUE TITLE
HUE group - dimmed start

### ISSUE THREAD

**Issue by Remme010** [hive/com.groups#28]
After the upgrade the groups can be easily created. Thanks for the great app created!!

 I have one group with 3 HUE lights.
When turning on the group, one light (random) stays on the lowest dim position possible.
Other two are normal turned as with last saved setting in app.

Homey 2.0.0
app. 2.0.2.246
IOS
group app. 2.1.707

**Comment by hive** [comment]
Are you able to restart the app - turn on the group - verify that one (random) light is at lowest dim level and then create a diagnostics report (More -> Apps -> Group -> (cog wheel)  -> Create diagnostic report) 

And ill be able to look into the issue in more detail. 

Thanks

**Comment by Remme010** [comment]
thanks Jamie for lloking into this. 

daignostic code is: e2208719-3f51-42f9-b632-176f60a44df8

**Comment by hive** [comment]
Ok, 

That didn't work - 

```
Crash report:

    Note: this log has been manually submitted by a user.
Log ID: e2208719-3f51-42f9-b632-176f60a44df8

---- stdout ----
2019-01-16 18:38:40 [log] [Groups] <Groups>
---- stderr ----
  ```

**Comment by Remme010** [comment]
the problem seems to be solved with the 2.1.708 update of last night!
Thanks for your help Jamie.
